atlascli.md

```diff
@@ -0,0 +1,66 @@
+# AtlasCLI (Pre-Release)
+
+The MongoDB AtlasCLI is a modern command line interface that enables you to manage MongoDB Atlas from the terminal.
+
+![atlascli-atlas-quickstart](https://user-images.githubusercontent.com/5663078/156184669-57c8ddce-6f0a-4e84-9311-2d996cb27942.gif)
+
+## Installing
+
+AtlasCLI is currently in the pre-release phase, do not use it in the production environment.
+
+### Pre-built Binaries
+
+Download the appropriate version for your platform from [AtlasCLI releases](https://github.com/mongodb/mongocli/releases).
+After you download the library, you can run it from anywhere and don't need to install it into a global location.
+This works well for shared hosts and other systems where you don't have a privileged account.
+
+Ideally, you should place this binary somewhere in your `PATH` for ease of use.
```

Comment:
"It would be best to place this binary somewhere in your `PATH` for ease of use. " sounds more friendly

Reply:
I will leave this comment open to see what doc team think about this chage

Reply:
I think either is fine, or leave off "Ideally" and just say "You can place this binary somewhere in your `PATH` for ease of use."

Reply:
Updated